tools/picard/picard_macros.xml

```diff
@@ -17,7 +17,9 @@
     </xml>
 
     <token name="@symlink_element_identifier@"><![CDATA[
-    ln -f -s '$inputFile' '$inputFile.element_identifier' &&
+    #import re
+    #set escaped_element_identifier = re.sub('[^\w\-_]', '_', str($inputFile.element_identifier))
```

Comment:
The underscore should already be included in `\w`, see https://docs.python.org/2/howto/regex.html#matching-characters